src/mlpack/tests/activation_functions_test.cpp

```diff
@@ -991,6 +1039,22 @@ TEST_CASE("CELUFunctionTest", "[ActivationFunctionsTest]")
   CheckCELUDerivativeCorrect(desiredActivations, desiredDerivatives);
 }
 
+/**
+ * Basic test of the ISRLU activation function.
+ */
+TEST_CASE("ISRLUFunctionTest", "[ActivationFunctionsTest]")
+{
+  const arma::colvec desiredActivations("-0.89442719 3.2 4.5 \
```

Comment:
Where did you get/calculate these test values?

Reply:
https://arxiv.org/abs/1710.09967 This is  the paper.   The   input value is same for all activation test cases which can be found at the beginning of the file and the desired activation can  be calculate          using the formula in the paper.     I just  used                a high       precision calculator and  calculated the values.